locale/en.json

```diff
@@ -94,6 +96,6 @@
   "none": "none",
   "remove": "remove",
   "close": "close",
-  "deprecation": "This node is deprecated, and will be out of support soon. More information under <a href='https://openwrt.org/supported_devices/432_warning'>4/32 warning</a>.<br>If you're the owner, please replace it with an modern device!",
+  "deprecationStatus": "This node is deprecated, and will be out of support soon. More information under <a href='https://openwrt.org/supported_devices/432_warning'>4/32 warning</a>.<br>If you're the owner, please replace it with an modern device!",
```

Comment:
Where was "deprecation" used before, as i do not see a usage rename anywhere in this PR

Reply:
Good catch.
Sorry, i added the deprecation lable as "deprecation" before, had an error somewhere and did find and replace but renamed this occurance too much.
I did not see it in my tests, because i have a custom deprecation message.